client/src/css/ClassCard.scss

```diff
@@ -1,18 +1,27 @@
+@import '../styles/colors.scss';
+
 .class-card {
   flex: none;
   width: 175px !important;
+  box-shadow: none !important;
+  border-radius: 15px !important;
+  margin: 0% !important;
 
   .card-content-container {
-    display: flex;
-    justify-content: space-evenly;
+    display: flex !important;
+    justify-content: space-between !important;
   }
 
   .delete-button {
-    height: 25px;
-    width: 25px;
+    height: 25px !important;
+    width: 50px !important;
+    font-family: Tommy !important;
+    padding: 0px;
   }
 
   .class-text {
-    font-size: 200%;
+    transition: all 0.3s ease !important;
+    font-family: Tommy !important;
+    font-size: 150%;
   }
```

Comment:
Can you put this css class inside the .card-content-container class? With scss, we can nest css rules, and since in ClassCard.jsx, `class-text` is inside `card-content-container`, it can go inside it in the scss file too!

Reply:
Gotcha!